adafruit_veml6070.py

```diff
@@ -145,18 +145,18 @@ def __init__(self, i2c_bus, _veml6070_it="VEML6070_1_T", ack=False):
 
 
     @property
-    def read(self):
+    def raw_uv(self):
         """
         Reads and returns the value of the UV intensity.
         """
-        read_buf = bytearray(2)
+        uv_buf = bytearray(2)
```

Comment:
This buffer is just used locally and could be named anything. Naming if just `buf` or `buffer` might be better visually, so it doesn't get confused with the property name.